includes/promoted-jobs/class-wp-job-manager-promoted-jobs-api.php

```diff
@@ -175,8 +175,8 @@ private function prepare_item_for_response( WP_Post $item ) {
 			'job_type'     => $terms_array,
 			'salary'       => [
 				'salary_amount'   => get_post_meta( $item->ID, '_job_salary', true ),
-				'salary_currency' => get_post_meta( $item->ID, '_job_salary_currency', true ),
-				'salary_unit'     => get_post_meta( $item->ID, '_job_salary_unit', true ),
+				'salary_currency' => get_the_job_salary_currency( $item ),
+				'salary_unit'     => get_the_job_salary_unit_display_text( $item ),
```

Comment:
Not strictly from this review, but it looks like these property names are not aligned with the ones in `docs/api/internal.json`. I have a preference for: `amount`, `currency` and `unit` versus `salary_*` counterparts, since these are already properties of "salary".

Reply:
Hey, good catch! I sent #2550 to fix this, as the purpose of that change is quite different from this PR (although both are related to salary!) 

Thanks!